.github/workflows/ci.yaml

```diff
@@ -77,6 +77,10 @@ jobs:
           toolchain: stable
           override: true
       - uses: Swatinem/rust-cache@v1
+      - name: Clippy and format
+        run: |
+          cargo clippy --all-features --all-targets -- -D warnings
+          cargo fmt -- --check
```

Comment:
Does Github shell run with `-e`? Would this step fail if only clippy fails? Why not have 2 steps? One for clippy and one for format.

Reply:
I would prefer clippy and rustfmt separate from each other, and clippy for each platform
Example: https://github.com/lapce/lapce/actions/runs/6130621751